core/src/main/resources/messages/messages.properties

```diff
@@ -18,6 +18,8 @@ message.buddy.device.vpn.disconnect=User disconnected VPN on device ''{0}''
 message.temp.nickname={0} {1}
 message.self.nickname={0} (me)
 message.system.nickname=Yona
+message.alternative.first.name=<{0}>
```

Comment:
Does this show `<Nickname>` literally? I hope the app did their html escaping right...

Reply:
Indeed, it shows something like ``<BR>``. The app doesn't do HTML, so that's not an issue.